src/uct/Makefile.am

```diff
@@ -149,6 +149,16 @@ noinst_HEADERS += \
 	ib/cm/cm.h
 endif
 
+if HAVE_RDMACM
+libuct_la_CPPFLAGS += $(RDMACM_CPPFLAGS)
+libuct_la_LDFLAGS  += $(RDMACM_LDFLAGS) $(RDMACM_LIBS)
+libuct_la_SOURCES  += \
+	rdmacm/rdmacm_md.c
```

Comment:
rdmacm should be part of 'ib'
both in configure/automake macros and in the directory structure
since it really requires verbs in order to work